app/upgrades/v26/upgrades.go

```diff
@@ -15,14 +36,33 @@ func CreateUpgradeHandler(
 	bpm upgrades.BaseAppParamManager,
 	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
-	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+	return func(context context.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		ctx := sdk.UnwrapSDKContext(context)
 		// Run migrations before applying any other state changes.
 		// NOTE: DO NOT PUT ANY STATE CHANGES BEFORE RunMigrations().
 		migrations, err := mm.RunMigrations(ctx, configurator, fromVM)
 		if err != nil {
 			return nil, err
 		}
 
+		// UNFORKING v2 TODO: I think there is just one new gov param that is not registered, which is why this is needed. Need to figure out what it is rather than re-setting all params.
+		// Set all gov params explicitly. E2E had issues when this was not done, so setting this here to ensure no issues on mainnet.
+		var newGovParams govv1.Params
+		if ctx.ChainID() == mainnetChainID || ctx.ChainID() == edgenetChainID {
+			newGovParams = govv1.NewParams(sdk.NewCoins(sdk.NewCoin("uosmo", osmomath.NewInt(1600000000))), sdk.NewCoins(sdk.NewCoin("uosmo", osmomath.NewInt(5000000000))), time.Second*1209600, time.Second*432000, time.Second*86400,
+				"0.200000000000000000", "0.500000000000000000", "0.667000000000000000", "0.334000000000000000", "0.250000000000000000", "0.500000000000000000", "", false, false, true, "0.010000000000000000")
```

Comment:
Just noting a section about gov for reference:
https://github.com/cosmos/cosmos-sdk/blob/main/UPGRADING.md#xgov-1

Given that SDK introduced expedited proposals, and we have a custom implementation, how is this handled?